app/views/content/landing/how-much-do-teachers-get-paid/_collage.html.erb

```diff
@@ -9,8 +9,8 @@
 
         <%= render Content::ChecklistCollageComponent.new(
           checklist: [
-            "You'll earn at least £30k (or more in London) if you're just starting as a teacher.",
-            "The average salary for a classroom teacher in England is over £40k a year."
+            "You'll earn at least #{ v :salaries_starting_minshortened } (or more in London) if you're just starting as a teacher.",
```

Comment:
@martyn-w Please could you fix the linting errors on lines 12 and 13 - this is a different format from usual so I'm unsure how to address them